### DISCUSSION TITLE
Add SSL/TLS to Email Settings

### DISCUSSION THREAD

**Discussion by Blarnix** [Peppermint-Lab/peppermint#348] [Ideas]
I'm trying to use iCloud custom domains (I need to switch, I know), but right now, I cannot log in without TLS or SSL. I'm not sure if it is supported by Peppermint but it is required to log into my email, it would be nice to be able to toggle. Thanks.

**Comment by Blarnix** [comment]
I now see that TLS is a possibiliy, but SSL I'm definitely looking for. Thanks.

**Comment by potts99** [comment]
is this for imap?
